<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RateMoveWindow</class>
 <widget class="QDialog" name="RateMoveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>влево</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите, куда будете ходить фишкой (номер фишки)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>вверх</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>вправо</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>вниз</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
